crates/astria-sequencer/src/transaction/mod.rs

```diff
@@ -42,6 +44,19 @@ pub(crate) async fn execute<S: StateWriteExt>(
         .await
 }
 
+#[derive(Debug)]
+pub(crate) struct InvalidNonce(pub(crate) u32);
+
+impl Display for InvalidNonce {
```

Comment:
Import `std::fmt` above, and then refer to all the types in here as `fmt::{Display, Formatter, Result}` for consistency.